packages/opentelemetry-cloud-monitoring-exporter/src/transform.ts

```diff
@@ -119,6 +129,57 @@ export function createTimeSeries(
   };
 }
 
+function transformResource(
+  resource: Resource,
+  projectId: string
+): MonitoredResource {
+  const templateResource = getTypeAndMappings(resource);
+  const type = templateResource.type;
+  const labels: { [key: string]: string } = { project_id: projectId };
+
+  for (const key of Object.keys(templateResource.labels)) {
+    if (!resource.labels[templateResource.labels[key]]) {
+      return { type: 'global', labels: { project_id: projectId } };
+    } else {
```

Comment:
Optional: Because you return in the if block above, consider removing the `else`. Doing so will flatten your structure as well.